vars/tagRemoteContainerImage.groovy

```diff
@@ -22,14 +23,32 @@ def call(Map params) {
         }
 
         if (deleteOriginalImage) {
+            
+            final String[] imageNameParts = sourceImage.split("/|:")
+            final Map image = [
+                registry: imageNameParts[0],
+                org: imageNameParts[1],
+                repo: imageNameParts[2],
+                tag: imageNameParts[3]
+            ]
             retry(3) {
-                sh """
-                    skopeo delete \
-                        --creds ${env.REGISTRY_CREDENTIALS} \
-                        docker://${sourceImage} \
-                    || sleep 10
-                """
+                if (image.registry == "quay.io") {
```

Comment:
Do you think that it would be useful to put a comment here to explain why this is needed?

Also, just to be sure, is the `REGSITRY_TOKEN` credential used here a long-lived token, or is it subject to expiration?

Reply:
> Do you think that it would be useful to put a comment here to explain why this is needed?

yes, good point, I'll add it

> Also, just to be sure, is the `REGSITRY_TOKEN` credential used here a long-lived token, or is it subject to expiration?

they're [long lived](https://docs.quay.io/api/#oauth-2-access-tokens)